docs/docsource/installation/install/install_xdm.rst

```diff
@@ -59,20 +59,20 @@ Deploy XDM Operator
 
 #. Install XDM Base:
 
-   .. code::
+   .. parsed-literal::
 
       /srv/scality/metalk8s-2.6.0/solutions.sh import --archive $ZENKO_BASE_ISO
-      sed "s/SOLUTION_ENV/zenko/g" /srv/scality/zenko-base-2.0.0-beta.3/operator.      yaml | kubectl apply -f -
+      sed "s/SOLUTION_ENV/zenko/g" /srv/scality/zenko-base-|version|-beta.3/operator.yaml | kubectl apply -f -
       kubectl -n zenko rollout status --timeout 10m deploy kubedb-operator
-      sed "s/SOLUTION_ENV/zenko/g" /srv/scality/zenko-base-2.0.0-beta.3/operator.      yaml | kubectl apply -f -
+      sed "s/SOLUTION_ENV/zenko/g" /srv/scality/zenko-base-|version|-beta.3/operator.yaml | kubectl apply -f -
```

Comment:
```suggestion
      sed "s/SOLUTION_ENV/zenko/g" /srv/scality/zenko-base-|version|/operator.yaml | kubectl apply -f -
```

Reply:
"-beta.3" is part of the version number. Replace these globally (carefully!)

Reply:
This appears to be present in all subsequent edits. 

In Zenko development, they use a mess of alpha- and beta- tags. These never see the light of day as far as customers are concerned. If you need testability, set the version variable to "2.0.0-beta.3" (or whatever) and this will update wherever |version| is in the docs.

Reply:
According to the projects_common_vars.py file, "The full version, including alpha/beta/rc tags" is "release = '2.0.0'" so I removed the version suffix and let |version| where it was necessary.